src/test/java/com/list/todo/services/TodoListServiceTest.java

```diff
@@ -38,22 +36,37 @@
     @Mock
     private UserService userService;
 
+    @Mock
+    private TagTaskKeyService tagTaskKeyService;
+
     @Mock
     private Pageable pageable;
 
     @InjectMocks
     private TodoListService todoListService;
 
+    private static final Long USER_ID = 1L;
+    private static final String USER_USERNAME = "username";
+    private static final Long SECOND_USER_ID = 2L;
+    private static final String SECOND_USER_USERNAME = "username2";
+    private static final Long TODO_LIST_ID = 3L;
+    private static final String TODO_LIST_NAME = "todoListName";
+    private static final String TODO_LIST_COMMENT = "comment";
+    private static final Long TODO_LIST_ID_2 = 4L;
+    private static final String TODO_LIST_NAME_2 = "todoListName2";
+    private static final Long TAG_ID = 5L;
+    private static final Long TAG_ID_2 = 6L;
+
     @Test
-    public void getTodoListById_Successful_ReturnsOptionalOfTodolist() {
+    public void getTodoListById_GetExistentTodoList_OptionalOfTodoList() {
         // arrange
-        long todoListId = 1;
-        TodoList todoList = new TodoList();
-        todoList.setTodoListName("name");
-        when(todoListRepository.findById(todoListId)).thenReturn(Optional.of(todoList));
+        TodoList todoList = ObjectsProvider.createTodoList();
```

Comment:
it would be nice to use static imports for ObjectsProvider methods, as well as for Assert. methods actually.